default.cfg

```diff
@@ -17,6 +17,11 @@
 # Optional: the frequency of checking the status of push notification (seconds)
 ; rest_poll_interval=1
 
+[starling_auto_provision]
```

Comment:
Add comment that: Auto provisioning is turned on by setting both emal and phone attribute bellow.